pkg/server/observer_test.go

```diff
@@ -348,8 +348,8 @@ func TestObserverServer_GetFlowsBetween(t *testing.T) {
 		t.Errorf("GetFlowsBetween error = %v, wantErr %v", err, nil)
 	}
 
-	for i := 0; i < 4; i++ {
-		assert.Equal(t, int64(6-i), got[i].GetFlow().Time.Seconds)
+	for i := 0; i < 6; i++ {
+		assert.Equal(t, int64(i+2), got[i].GetFlow().Time.Seconds)
```

Comment:
This change in the test doesn't scream at me "the order is now reversed"

Reply:
Does it need to?

Reply:
Ideally tests are easy to read and reason about, yes. This is more of a comment on the test setup though